chains.yaml

```diff
@@ -871,20 +871,12 @@
   dockerfile: cargo
```

Comment:
We might be able to change this to `cosmos` now without the shared library requirement so that we get a fully static binary